site/src/Controller/NotificationController.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Notification;
+use Mgilet\NotificationBundle\NotifiableInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Mgilet\NotificationBundle\Manager\NotificationManager;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Class NotificationController
+ */
+class NotificationController extends AbstractController
+{
+    /**
+     * @var NotificationManager
+     */
+    protected $notificationManager;
+
+    /**
+     * NotificationController constructor.
+     *
+     * @param NotificationManager $notificationManager
+     */
+    public function __construct(NotificationManager $notificationManager)
+    {
+        $this->notificationManager = $notificationManager;
+    }
+
+    /**
+     * Mark a Notification as seen
+     *
+     * @Route("/notification/seen", name="notification_mark_seen")
```

Comment:
Will Notification Inbox be a part of User Controller?
SInce routes will now be changed to /user/notification
If that is the case, are we going to move the logic to userController?

Reply:
That would make sense, yes.